### DISCUSSION TITLE
Need Help in Running oppia android

### DISCUSSION THREAD

**Discussion by Ayush-865** [oppia/oppia-android#4975] [Q&A (Installation)] [answered]
### Operating System

windows 11

### Android Studio Version

electric eil

### RAM size

8

### Free disk space

300gb

### Which step are you stuck on?

I want to run the android app on my phone
I am not able to
I am new and unable to understand what to do now 
There is no sync gradle option also the run button is not working
I have connected my physical phone with android studio
![Screenshot (132) - Copy](https://github.com/oppia/oppia-android/assets/126230501/2ffd8124-2bee-4df8-a85a-9bb0ae171634)
![image](https://github.com/oppia/oppia-android/assets/126230501/c8e66006-2534-4109-9cd5-7b96b7b65104)
![image](https://github.com/oppia/oppia-android/assets/126230501/835ee16d-3e48-4e19-b0fc-a5ba06351813)
![Screenshot (135)](https://github.com/oppia/oppia-android/assets/126230501/f765fe10-d923-4386-a95d-5c43b6468f9a)

### Error Log

```bash
Please help
```

**Comment by InquisitiveCatherine** [comment, marked as answer]
Hi could you open C:\OPPIA_ANDROID\oppia-android. See if that helps.
It looks like you are on parent directory of the project.
![image](https://github.com/oppia/oppia-android/assets/133565952/1b784989-3014-4264-8cb8-d88e645eba15)

**Comment by Ayush-865** [comment]
My issue is solved. I was in the parent directory. I dint knew about. But now i am good to go

**Comment by Ayush-865** [comment]
Thanks a lot !!!
